scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/ManageUsers.java

```diff
@@ -0,0 +1,579 @@
+    }
+
+    private static String getSchedulerFile(String path) {
+        String schedulerFile = path;
+        if (!(new File(schedulerFile).isAbsolute())) {
+            //file path is relative, so we complete the path with the prefix Scheduler_Home constant
+            schedulerFile = PASchedulerProperties.SCHEDULER_HOME.getValueAsString() + File.separator + path;
+        }
+        return schedulerFile;
+    }
+
+    private static void displayHelp(Options options) {
+        HelpFormatter hf = new HelpFormatter();
+        hf.setWidth(135);
+        hf.printHelp("proactive-users" + Tools.shellExtension(), "", options, "", true);
+    }
+
+    enum Action {
+        CREATE("Creating User."),
+        UPDATE("Updating existing user"),
+        DELETE("Deleting user");
+        final String message;
+
+        Action(String message) {
+            this.message = message;
+        }
+
+        public static Action getAction(CommandLine cmd) {
+            if (cmd.hasOption(CREATE_OPTION_NAME)) {
+                return Action.CREATE;
+            } else if (cmd.hasOption(UPDATE_OPTION_NAME)) {
+                return Action.UPDATE;
+            } else if (cmd.hasOption(DELETE_OPTION_NAME)) {
+                return Action.DELETE;
+            } else {
+                throw new IllegalArgumentException("Command line does not contain, create, update or delete action: " + cmd);
+            }
+        }
+
+        public boolean isCreate() {
+            return this == Action.CREATE;
+        }
+
+        public boolean isUpdate() {
+            return this == Action.UPDATE;
+        }
+
+        public boolean isDelete() {
+            return this == Action.DELETE;
+        }
+
+        @Override
+        public String toString() {
+            return message;
+        }
+    }
+
+    static class UserInfo {
+        private String login;
+        private String password;
+        private String[] groups;
+
+        public UserInfo() {
+        }
+
+        public UserInfo(String login, String password, String[] groups) {
+            this.login = login;
+            this.password = password;
+            this.groups = groups;
+        }
+
+        public String getLogin() {
+            return login;
+        }
+
+        public void setLogin(String login) {
+            this.login = login;
+        }
+
+        public String getPassword() {
+            return password;
+        }
+
+        public void setPassword(String password) {
+            this.password = password;
+        }
+
+        public String[] getGroups() {
+            return groups;
+        }
+
+        public void setGroups(String[] groups) {
+            this.groups = groups;
+        }
+    }
+
+    static class ManageUsersException extends Exception {
+        public String getAdditionalInfo() {
+            return additionalInfo;
+        }
+
+        private String additionalInfo = null;
+
+        public ManageUsersException(String message) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1213)